lms/templates/discussion/_recent_active_posts.html

```diff
@@ -11,7 +11,7 @@
     </header>
     <ol class="discussion-sidebar-following-list">
       % for thread in recent_active_threads:
-        <li><a href="${helpers.permalink(thread) | h}"><span class="sidebar-following-name">${thread['title'] | h}</span> <span class="sidebar-vote-count">${thread['votes']['point'] | h}</span></a></li>
+        <li><a href="${permalink(thread) | h}"><span class="sidebar-following-name">${thread['title'] | h}</span> <span class="sidebar-vote-count">${thread['votes']['point'] | h}</span></a></li>
```

Comment:
This doesn't happen to fix https://openedx.atlassian.net/browse/TNL-2919, does it? Any way to work that fix into this PR (without spending more than 30 minutes on it)?